integrated_channels/integrated_channel/exporters/learner_data.py

```diff
@@ -87,22 +87,6 @@ def grade_audit(self):
         """
         return self.GRADE_AUDIT
 
-    def fetch_course_details(self, course_details, enrollment_course_id):
```

Comment:
this caching was not being used so simplified it by just calling the new method now